draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -475,9 +475,157 @@ Additional JWT VC Issuer Metadata configuration parameters MAY also be used.
 
 ## JWT VC Issuer Metadata Validation
 
-The `issuer` value returned MUST be identical to the `iss` value of the JWT. If
-these values are not identical, the data contained in the response MUST NOT be
-used.
+The `issuer` value returned MUST be identical to the `iss` value of the
+JWT. If these values are not identical, the data contained in the response
+MUST NOT be used.
+
+# Type Metadata {#type-metadata}
+
+A type is associated with metadata defining, for example, information about the type, a schema defining which claims MAY or MUST appear in the SD-JWT VC, and how they can be displayed.
```

Comment:
This is qualified with "for example" but the type metadata in the PR currently doesn't define this stuff. I'm still unsure about leaving portions of sd-jwt-vc-types like claims and display out of this PR. But should we at least track that in an issue?

Reply:
created #224